docs/_docs/templates.md

```diff
@@ -178,7 +178,7 @@ you come up with your own tags via plugins.
     <tr>
       <td>
         <p class="name"><strong>Group By Expression</strong></p>
-        <p>Group an array's items using a Liquid expression.</p>
+        <p>Group an array's items using a Liquid expression. Jekyll v3.4.0 & later.</p>
```

Comment:
@karel-3d we have an include for that:
`{% include docs_version_badge.html version="3.4.0" %}`

Reply:
I copy-pasted this from few lines earlier, where_exp. Should I change both?

Reply:
yes, please.